docs/vm.md

```diff
@@ -53,6 +52,7 @@ Commands:
   ops             Dump opcodes of the current loaded program
   parse           Parse provided argument and convert it into other possible formats
   run             Execute the current loaded script
+  slots           Show slots (static, local and arguments) contents
```

Comment:
Please move it to separate PR, we need something more like `slot X` here, dumping all slots at the same time is not very useful.